packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -271,6 +271,11 @@ export const makeTreeEditGenerator = (
 						.integer(Number.MIN_SAFE_INTEGER, Number.MAX_SAFE_INTEGER)
 						.toString(),
 				};
+			case "com.fluidframework.leaf.handle":
```

Comment:
The pattern of explicitly putting the type strings here is not our encouraged pattern. Instead use leaf.handle.name here (and below: note that it typed such that you don't need to brand it: having to brand something should be an indication that the code probably should be using an already branded value from somewhere).

Generally, type names should never need to be written out in code, and can always be referend from he schema they are referring to. It makes it clear what the referenced type is, and is more maintainable and more type safe.

Feel free to fix the nearby occurrences of this anti-pattern as well.

Reply:
i don't think this is the correct PR to refactor. that should be done as a follow up by the feature/test owners here if necessary.

Reply:
Yes, cleaning this up should be done in a separate PR. I'm ok with this staying how it is for consistency for now, but I wanted to point out that this is not how it should be done and this approach should not be applied elsewhere.

Reply:
For tracking: [AB#7878](https://dev.azure.com/fluidframework/internal/_workitems/edit/7878)